src/main/java/mate/academy/car/sharing/entity/Rental.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.car.sharing.entity;
+
+import java.time.LocalDate;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Getter
+@Setter
+@Entity
+@Accessors(chain = true)
```

Comment:
@Accessors(chain = true) - is redundand in Entity, because Hibernate manage this entity, so we don't need a chain of setters